src/Product.java

```diff
@@ -6,6 +6,7 @@
 
     //added attributes
     private int price;
+    private int amountToSale;
```

Comment:
Do we need to initialize price & amountToSal to 0? maybe It happens automatically :)

Reply:
You're absolutely right. We better initialize it ourselves!